addons/udes_get_info/models/stock_picking_type.py

```diff
@@ -0,0 +1,24 @@
+"""
+Stock Picking Type model.
+
+Picking types define the operations that can be performed in a warehouse.
+"""
+from odoo import models
+
+
+class StockPickingType(models.Model):
+    """Stock Picking Type model."""
+
+    _inherit = "stock.picking.type"
+
+    # These fields will be extracted by get_info().
+    _get_info_field_names = {
+        "code",
+        "count_picking_ready",
+        "default_location_dest_id",
+        "default_location_src_id",
+        "display_name",
+        "id",
+        "name",
+        "sequence",
```

Comment:
Perhaps `company_id` and `warehouse_id` too

Reply:
Added. please review.